modules/local/novaseq_err.nf

```diff
@@ -0,0 +1,84 @@
+process NOVASEQ_ERR {
+    tag "$meta.run"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconductor-dada2=1.22.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bioconductor-dada2:1.22.0--r41h399db7b_0' :
+        'quay.io/biocontainers/bioconductor-dada2:1.22.0--r41h399db7b_0' }"
+    input:
+    tuple val(meta), path(errormodel)
+
+    output:
+    tuple val(meta), path("*.md.err.rds"), emit: errormodel
+    tuple val(meta), path("*.md.err.pdf"), emit: pdf
+    tuple val(meta), path("*.md.err.log"), emit: log
+    tuple val(meta), path("*.md.err.convergence.txt"), emit: convergence
+
+    script:
+    if (!meta.single_end) {
+        """
+        #!/usr/bin/env Rscript
+        suppressPackageStartupMessages(library(dada2))
+
+        errF = readRDS("${errormodel[0]}")
+        errR = readRDS("${errormodel[1]}")
+
+        #monotone decreasing
+        sink(file = "${meta.run}.md.err.log")
+        make.monotone.decreasing <- function(v) sapply(seq_along(v), function(i) max(v[i:length(v)]))
+
+        errF.md <- t(apply(getErrors(errF), 1, make.monotone.decreasing))
+        errF.md.full <- errF
+        errF.md.full$err_out <- errF.md
```

Comment:
This doesnt work, change to:
```suggestion
        errF.md.full\$err_out <- errF.md
```
explanation: 
`$` in a nextflow script block is interpreted as nextflow variable, i.e. in `errF = readRDS("${errormodel[0]}")`. If you want to use the R style `$` it needs to be excaped, i.e. `\$`.
To avoid this (i.e. using 1:1 R script code), you could make an R script in `bin/` and just feed the R script here with data.